player/clipboard/clipboard-wayland.c

```diff
@@ -172,6 +178,59 @@ static const struct ext_data_control_device_v1_listener data_device_listener = {
     data_device_handle_primary_selection,
 };
 
+static void data_source_handle_send(void *data,
+                                    struct ext_data_control_source_v1 *data_source,
+                                    const char *mime_type,
+                                    int32_t fd)
+{
+    struct clipboard_wayland_seat *s = data;
+    struct clipboard_wayland_priv *wl = s->wl;
+    if (strcmp(mime_type, "text/plain;charset=utf-8") == 0 ||
+        strcmp(mime_type, "text/plain") == 0 ||
+        strcmp(mime_type, "text") == 0)
+    {
+        struct pollfd fdp = { .fd = fd, .events = POLLOUT };
+        if (poll(&fdp, 1, 0) <= 0)
+            goto cleanup;
```

Comment:
We don't need this right? If there's an error, `POLLERR` and `POLLHUP` should catch it.